client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx

```diff
@@ -62,67 +62,70 @@ const TabManagement: React.FC<Props> = (tabManagementProps: Props): JSX.Element
     const formsValidations = useSelector<StoreStateType, any>((state) => state.formsValidations[investigationId]);
 
-    const isTabInValid = (tabId: number) => {
-        return formsValidations !== undefined && formsValidations[tabId] !== null && !formsValidations[tabId]  
+    const isTabValid = (tabId: number) => {
+        return formsValidations !== undefined && formsValidations[tabId] !== null && !formsValidations[tabId];
     }
     
     useEffect(() => {
-        !areThereContacts && setFormState(investigationId, last.id, true);
+        !areThereContacts && setFormState(investigationId, lastTab.id, true);
     }, [areThereContacts]);
 
     return (
         <Card className={classes.card}>
-                <Tabs
-                    value={currentTab}
-                    indicatorColor='primary'
-                    textColor='primary'
-                >
-                     {
-                        tabs.map((tab: TabObj) => 
-                            <StyledTab 
-                                // @ts-ignore
-                                type='submit'
-                                form={`form-${currentTab}`}
-                                onClick={() => {setNextTab(tab.id)}}
-                                key={tab.id}
-                                label={tab.name}
-                                icon={isTabInValid(tab.id) ? <ErrorOutlineIcon/> : undefined}
-                                className={isTabInValid(tab.id) ? classes.errorIcon : undefined}
-                            />
-                    )}
-                    {areThereContacts && <StyledTab 
-                                // @ts-ignore
-                                type='submit'
-                                form={`form-${currentTab}`}
-                                onClick={() => {setNextTab(last.id)}}
-                                key={last.id}
-                                label={last.name}
-                                icon={isTabInValid(last.id) ? <ErrorOutlineIcon/> : undefined}
-                                className={isTabInValid(last.id) ? classes.errorIcon : undefined}
-                            />
-                    }
-                </Tabs>
-                <div className={classes.displayedTab}>
-                    {currentTab == 4 ? last.displayComponent:tabs[currentTab].displayComponent}
-                </div>
+            <Tabs
+                value={currentTab}
+                indicatorColor='primary'
+                textColor='primary'
+            >
+                {
+                    tabs.map((tab: TabObj) =>
+                        <StyledTab
+                            // @ts-ignore
+                            type='submit'
+                            form={`form-${currentTab}`}
+                            onClick={() => { setNextTab(tab.id) }}
+                            key={tab.id}
```

Comment:
you can try key={areThereContacts  ? lastTab.id :  tab.id} instead of duplicate StyledTab

Reply:
removed duplicate tab